### PR TITLE
[#449] Add more fields to CSV download

### PR DESCRIPTION
**Pull request by robredpath**

### CODE REVIEW

**Comment by robredpath** [comment]
@BobHarper1 @morchickit this PR adds the fields that Bob suggested in #449

File: grantnav/csv_layout.py [hunk 1]
```diff
@@ -1,145 +1,89 @@
...
+    ("Data Source", "dataset.distribution.0.downloadURL"),
+    ("Publisher:Name", "dataset.publisher.name"),
+    ("Recipient Region", "result.recipientRegionName"),
+    ("Recipient District", "result.recipientDistrictName"),
+    ("Recipient District Geographic Code", "result.recipientDistrictGeoCode"),
+    ("Recipient Ward", "result.recipientWardName"),
+    ("Recipient Ward Geographic Code", "result.recipientWardNameGeoCode"),
+    ("Retrieved for use in GrantNav", "dataset.datagetter_metadata.datetime_downloaded"),
+    ("License (see note)", "dataset.license"),
+    ("Note,this file also contains OS data © Crown copyright and database right 2016, Royal Mail data © Royal Mail copyright and Database right 2016, National Statistics data © Crown copyright and database right 2015 & 2016, see http://grantnav.threesixtygiving.org/datasets/ for more information.", ""),
```
Comment (Bjwebb) [inline grantnav/csv_layout.py]: "Note,this" - I think we're missing a space here

**Comment by morchickit** [comment]
Let's do this!

**Comment by robredpath** [comment]
This is now live

**Pull request merged**